extensions/vscode/src/eventErrors.test.ts

```diff
@@ -0,0 +1,72 @@
+// Copyright (C) 2024 by Posit Software, PBC.
+
+import { describe, expect, test } from "vitest";
+import { EventStreamMessage } from "./api";
+import {
+  EventStreamMessageErrorCoded,
+  isCodedEventErrorMessage,
+  isEvtErrDeploymentFailed,
+  handleEventCodedError,
+} from "./eventErrors";
+import { ErrorCode } from "./utils/errorTypes";
+
+function mkEventStreamMsg(data: Record<PropertyKey, any>): EventStreamMessage;
+function mkEventStreamMsg(
```

Comment:
Little bit more complex up here, but it looks so much more usable in the tests themselves. Great work on the typing!